app/components/SessionExpiryHandler.tsx

```diff
@@ -3,7 +3,7 @@ import { useRouter } from 'next/router';
 import type { SessionExpiryHandlerQuery } from '../__generated__/SessionExpiryHandlerQuery.graphql';
 import { graphql, fetchQuery, useRelayEnvironment } from 'react-relay';
 import { SessionTimeoutHandler } from '@bcgov-cas/sso-react';
-import config from '../config.js';
+import config from '../config/index.js';
```

Comment:
Looks like we have a fix that was undone at some point. This front-end file should not load the `config`. It should use next.js' [`publicRuntimeConfig`](https://nextjs.org/docs/api-reference/next.config.js/runtime-configuration) instead